ethereum/test/test_mmr_verification.js

```diff
@@ -392,4 +415,32 @@ describe("MMRVerification Contract", function () {
             expect(await mmrVerification.verifyInclusionProof.call(root, leafNodeHash, proof.leaf_index, proof.leaf_count, proof.items)).to.be.true
         });
     })
-})
+
+    context("7-leaf MMR from fixture", function () {
```

Comment:
nit: seems like the fixture tests can replace the 7-leaf and 15-leaf tests above? Fine to leave as is for this PR though

Reply:
Yeah, good point – but more tests can't really hurt, can they?

Reply:
;) I'm leaving them in for now, since the data is different